### ISSUE TITLE
Payload validation for 0x01FF packet

### ISSUE THREAD

**Issue by Onl1ne1373** [zxdavb/ramses_cc#145]
The payload validation for 0x1FF in ramses_rf is:

```python
Code._01FF: {  # unknown_01ff, TODO: definitely a real code, Itho Spider
        SZ_NAME: "message_01ff",
        I_: r"^0[01][0-9A-F]{50}$",
        RQ: r"^0[01][0-9A-F]{50}$",
        W_: r"^00[0-9A-F]{50}$",
```

But in **ramses_cc** the payload validation is:
```python
vol.Required("payload"): cv.matches_regex(r"^[0-9A-F]{1,48}$")
```
So, sending `01FF` packages in Home Assistant fails due to the maximum length.

**Comment by zxdavb** [comment]
Can you provide more information?
 - why is its payload length 50 and not 48?
 - do you have any examples of such packets?

**Comment by zxdavb** [comment]
Sorry, I am being stupid.

There is a typo in **ramses_cc**, it should be:
```python
vol.Required("payload"): cv.matches_regex(r"^([0-9A-F][0-9A-F]){1,48}$"),
```

I have made this fix - it will be included within **0.31.4**.
